dsc/workflows/base/__init__.py

```diff
@@ -139,34 +141,68 @@ def submit_items(self, collection_handle: str) -> dict:
         Returns a dict with the submission results organized into succeeded and failed
         items.
         """
-        items: dict[str, Any] = {"succeeded": {}, "failed": {}}
+        logger.info(
+            f"Submitting messages to the DSS input queue '{CONFIG.dss_input_queue}' "
+            f"for batch '{self.batch_id}'"
+        )
+        submission_summary = {
+            "total": 0,
+            "submitted": 0,
+            "errors": 0,
+        }
+
+        items = []
         for item_submission in self.item_submissions_iter():
-            item_id = item_submission.item_identifier
+            submission_summary["total"] += 1
+            item_identifier = item_submission.item_identifier
+
             try:
                 item_submission.upload_dspace_metadata(self.s3_bucket, self.batch_path)
+            except Exception as exception:
+                logger.error(f"Failed to upload DSpace metadata for item. {exception}")
+                self.workflow_events.errors.append(str(exception))
+                submission_summary["errors"] += 1
+                continue
+
+            try:
                 response = item_submission.send_submission_message(
                     self.workflow_name,
                     self.output_queue,
                     self.submission_system,
                     collection_handle,
                 )
-            except Exception as exception:
-                logger.exception(f"Error processing submission: '{item_id}'")
-                items["failed"][item_id] = exception
+            except ClientError as exception:
+                logger.error(
+                    f"Failed to send submission message for item: {item_identifier}. "
+                    f"{exception}"
+                )
+                self.workflow_events.errors.append(str(exception))
+                submission_summary["errors"] += 1
                 continue
-            status_code = response["ResponseMetadata"]["HTTPStatusCode"]
-            if status_code != HTTPStatus.OK:
-                items["failed"][item_id] = RuntimeError("Non OK HTTPStatus")
+            except Exception as exception:
+                logger.error(
+                    f"Unexpected error occurred while sending submission message "
+                    f"for item: {item_identifier}. {exception}"
+                )
```

Comment:
[IN PROGRESS]
These changes emerged from a desire to provide more granularity to any exceptions raised from the call to `item_submission.send_submission_message`, which calls the `SQS.Client.send_message`